diff --git a/a/Rendering/OpenGL2/vtkWin32OpenGLRenderWindow.cxx b/b/Rendering/OpenGL2/vtkWin32OpenGLRenderWindow.cxx
index 6cbf334..583d28d 100644
--- a/a/Rendering/OpenGL2/vtkWin32OpenGLRenderWindow.cxx
+++ b/b/Rendering/OpenGL2/vtkWin32OpenGLRenderWindow.cxx
@@ -1139,7 +1139,7 @@ void vtkWin32OpenGLRenderWindow::DestroyWindow()
 int *vtkWin32OpenGLRenderWindow::GetSize(void)
 {
   // if we aren't mapped then just return the ivar
-  if (this->Mapped)
+  if (this->Mapped && !this->OffScreenRendering)
   {
     RECT rect;
 
